client/src/components/productDetails/ProductInformation.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import Price from './Price.jsx';
+
+function ProductInformation({ product, reviews, selectedStyle }) {
+  let ratingTotal = 0;
+  for (let i = 0; i < reviews.length; i++) {
+    ratingTotal += reviews[i].rating;
+  }
+  const avgRating = ratingTotal / reviews.length;
+
+  return (
+    <div>
+      Title:
```

Comment:
Splitting evaluations up on multiple lines results in multiple text elements being rendered. Try using template literals to combine text into one evaluation.

Reply:
Thanks so much Kate!! Will do!